v1/k8s/namespace/white-listed-pod.rego

```diff
@@ -0,0 +1,79 @@
+package verify
```

Comment:
We should have a way to specify "all namespaces", I did not understand this is suppported (from the code)